### ISSUE TITLE
Error when Update Packal

### ISSUE THREAD

**Issue by nntoan** [shawnrice/packal-updater#4]
Hello,

Packal Updater said: "Error manifest workflow...."

I'm wondering why this going to be happen?

**Comment by shawnrice** [comment]
Can you give more details on the error?

**Comment by nntoan** [comment]
How can I do that?

**Comment by shawnrice** [comment]
Where are you seeing three error? Have you tried force clearing the manifests?

**Comment by nntoan** [comment]
I type `packal` in Alfred 2 and press <kbd>Return</kbd>. Then, I've got this:

```
Error: Packal Updater
The workflow manifest is not valid, and there is no valid Internet connection to retrieve a new one.
```

**Comment by shawnrice** [comment]
Okay. Are you in China? I've run into this problem before because of China blocking google. A better explanation for a quick fix for yourself is here:

http://www.alfredforum.com/topic/4579-packal-updater/page-4#entry34147

**Comment by nntoan** [comment]
No, of course not. I'm Vietnamese and we're not blocking Google. That's very strange. How could I debug this?

**Comment by tape88** [comment]
with same problem here
